locales/ja/LC_MESSAGES/user_guide.po

```diff
@@ -1541,18 +1540,18 @@ msgid ""
 "Metadata and Authorship section of this User Guide](../topics/metadata-and-"
 "authorship.md)."
 msgstr ""
-"CW定義にメタデータを与える例については、［このユーザーガイドのメタデータと著"
-"者名のセクション］(../topics/metadata-and-authorship.md)を参照してください。"
+"CW定義にメタデータを与える例については、 "
+"[このユーザーガイドのメタデータと著者名のセクション](../topics/metadata-and-"
+"authorship.md)を参照してください。"
 
 #: ../../src/topics/best-practices.md:40 ecf2b9c0b1664afe9c4e180610610021
-#, fuzzy
 msgid ""
 "Include [attribution information][license-example] for the author(s) of the "
 "CWL tool or workflow description. Use unambiguous identifiers like [ORCID]"
 "[orcid]."
 msgstr ""
-"CWLツールやワークフローの定義の著者の[帰属情報][ライセンス例]を含みます。[ORC"
-"ID][orcid]のような曖昧でない識別子を使用してください。"
+"CWLツールやワークフローの定義の著者の[帰属情報][license-example]を含みます。["
+"ORCID][orcid]のような曖昧でない識別子を使用してください。"
```

Comment:
In `[text][link_target]` the `text` should be translated, but not the `link_target`